sunpy/map/mapbase.py

```diff
@@ -2052,7 +2052,9 @@ def peek(self, draw_limb=False, draw_grid=False,
                 pad = 0.12  # Pad to compensate for ticks and axes labels
             else:
                 pad = 0.05  # Default value for vertical colorbar
-            figure.colorbar(im, pad=pad)
+            colorbar_label = str(self.unit) if self.unit is not None else ""
+            figure.colorbar(im, pad=pad).set_label(colorbar_label,
+                                                   rotation=0, labelpad=-50, y=-0.02, size=12)
```

Comment:
Does this need to be two lines?

Reply:
PEP 8 gives me an issue if the files aren't in 2 separate lines so